stacks/argocd/README.md

```diff
@@ -0,0 +1,49 @@
+Getting Started after launching your Argo CD 1-Click App
+
+After you have downloaded your kube config file, and are able to successfully connect to your DigitalOcean Kubernetes cluster (see https://cloud.digitalocean.com/kubernetes/clusters/ if you haven’t connected to your cluster) follow the instructions below to start using Argo CD.
+
+## Argo CD CLI
+
+Download the latest Argo CD version from https://github.com/argoproj/argo-cd/releases/latest.
+
+Also available in Mac Homebrew:
+
+`brew tap argoproj/tap
+brew install argoproj/tap/argocd`
+
+## Login using the Argo CD CLI
+
+Login as the admin user. The initial password is autogenerated to be the pod name of the Argo CD API server. This can be retrieved with the command:
+
+`kubectl get pods --all-namespaces`
+
+You'll see output similar to the following: 
+
+`yourhostname:.kube yourusername$ kubectl get pods --all-namespaces
```

Comment:
use triple ticks to preserve text formatting

Reply:
thanks - made this change